lib/Doctrine/ORM/Sequencing/Generator.php

```diff
@@ -11,11 +11,9 @@
     /**
      * Generates an identifier for an entity.
      *
-     * @param object $entity
-     *
-     * @return \Generator
+     * @return string|int
```

Comment:
What happened here? :|

Reply:
It was just wrong before, you can see based on the implementation...